docs/dts-tests.md

```diff
@@ -0,0 +1,44 @@
+# DTS tests
+
+This document describes current DTS tests coverage. Tests planned:
+
+* E2E tests;
+* Unit tests.
+
+## E2E tests
+
+E2E - End to End tests, tests that verify how DTS goes from start to end for
+every user workflow (e.g. installation, update, etc.) for every platform.
+
+Location in OSFV: `dts/dts-e2e.robot`.
+
+These tests include modifications for DTS platform emulation so the tests
+could be launched on Qemu. Then every test case choose the workflow by
+choosing DTS menu option, and provides credentials (if necessary). So, the
+start conditions are: platform configuration and workflow selection.
+
+Then every test case goes through the chosen workflow and checks for expected
+behavior. If everything goes as expected - test case passes, otherwise - it
+fails.
+
+Control variables:
+
+* `dts_ipxe_link`: useful if you are testing DTS which is not released yet. Just
+  put here a link to your script which will load your DTS. By default DTS is
+  being booted from `dl.3mdeb.com`;
+* `dpp_logs_key`, `dpp_download_key`, `dpp_password`: for DPP credentials, if
```

Comment:
Can we apply this suggestion here: https://github.com/Dasharo/meta-dts/pull/193/files#r1853692417 ?
Let me know your thoughts on this.

Reply:
This will be a problem for a short period of time, because soon, when we migrate our infrastructure to MinIO, we will be able to use one set of credentials for all subscriptions. So, I would rather stay with the way it is now.

Reply:
ok, let's see how short this period will be then